src/lib.rs

```diff
@@ -26,108 +26,115 @@
 //!
 //! ## Multiaddr
 //!
-//! A `Multiaddr` is a way to reach a node. Examples:
+//! A [`Multiaddr`] is a self-describing network address and protocol stack
+//! that is used to establish connections to peers. Some examples:
 //!
 //! * `/ip4/80.123.90.4/tcp/5432`
 //! * `/ip6/[::1]/udp/10560/quic`
-//! * `/unix//path/to/socket`
+//! * `/unix/path/to/socket`
```

Comment:
This extra `/` is actually on purpose. As far as I know, the specs of multiaddr don't specify a way to express these kind of addresses, but I remember seeing this as an example somewhere.

Reply:
Oh, good to know, thanks.